class-wc-settings-page-komoju.php

```diff
@@ -134,6 +168,62 @@ function komoju_woocommerce_reset_endpoint_field(event) {
 <?php
     }
 
+    // Action handler for rendering settings with type = 'komoju_setup_button'
+    public function output_setup_button($setting)
+    {
+        $nonce = wp_generate_uuid4();
+        update_option('komoju_woocommerce_nonce', $nonce);
+
+        $already_connected = get_option('komoju_woocommerce_secret_key') ? true : false;
+
+        $setup_url = KomojuApi::endpoint() . '/plugin/auth?' .
+            'post_url=' . rawurlencode($this->contracted_url_for_webhooks()) . '&' .
+            'webhook_url=' . rawurlencode($this->contracted_url_for_webhooks()) . '&' .
+            'nonce=' . rawurlencode($nonce); ?>
```

Comment:
This is a bit of a nitpick but I was a little confused about why we seemingly suddenly went from PHP to HTML over here.

Perhaps the indentation of the `<?php ... ?>` tag could be handled better? I had to look a couple times before I saw the `?>` at the end of the `$setup_url` declaration.

Maybe something more like this is cleaner?

```
<?php
  code code code
  code code code
  code code code
?>
<tr>
  html html html
</tr>
```

Feel free to treat this as non blocking since I see the same style in other places in the code. Changing it would probably be beyond the scope of this PR.

Reply:
Unfortunately, the position of the `?>` is actually dictated by the linter tool (php-cs-fixer) - I originally wrote this more like your suggestion. Perhaps there's a setting somewhere that can make the tool prefer better style :smile: but indeed that'd result in sweeping changes so will do in another PR.